raft.go

```diff
@@ -1667,9 +1667,13 @@ func (r *raft) handleAppendEntries(m pb.Message) {
 		hintIndex := min(m.Index, r.raftLog.lastIndex())
 		hintIndex = r.raftLog.findConflictByTerm(hintIndex, m.LogTerm)
```

Comment:
Wouldn't it be better for this to return the term, since we're just about to recompute it?

Reply:
Done.